plugins/formal-verification/testData/diagnostics/uniqueness/bad_unique/direct_pass_shared_to_unique.kt

```diff
@@ -0,0 +1,16 @@
+// UNIQUE_CHECK
+// NEVER_VALIDATE
```

Comment:
If `UNIQUE_CHECK` ensures no conversion happens, it should also ensure validation doesn't happen.